.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -41,6 +41,9 @@ jobs:
           globs: '**/*.md'
 
   build:
+    permissions:
```

Comment:
vcpkg still seems to be rebuilding, wonder if it's related to this part or something

Reply:
Yes, this seems to be the case, I've set permissions back to default, which is working again :)